### PR TITLE
URL Shortener Command 

### PR DESCRIPTION
**Pull request by HenryMartins**
Sorry anything, it's my first command in Switchblade.

I used Tinyurl's API.

### CODE REVIEW

File: src/commands/utility/tinyurl.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        type: 'string',
+        full: true,
+        missingError: 'commands:tinyurl.missingSentence'
+      }]
+    })
+  }
+
+  async run ({ t, author, channel }, text) {
+   channel.startTyping()
+   request(`http://tinyurl.com/api-create.php?url=${encodeURIComponent(text)}`, function (e, r, b) {
```
Comment (metehus) [inline src/commands/utility/tinyurl.js]: use node-fetch instead

**Comment by metehus** [comment]
also wait for #737 and then use the url parameter

File: src/commands/utility/tinyurl.js (hunk 1 above)
Comment (metehus) [reply src/commands/utility/tinyurl.js]: or snekfetch idk

Comment (henrythedeveloper) [reply src/commands/utility/tinyurl.js]: Done!

File: src/commands/utility/tinyurl.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        type: 'string',
+        full: true,
+        missingError: 'commands:tinyurl.missingSentence'
+      }]
+    })
+  }
+
+  async run ({ t, author, channel }, text) {
+   channel.startTyping()
+   fetch(`http://tinyurl.com/api-create.php?url=${encodeURIComponent(text)}`).then(shorturl => {
```
Comment (metehus) [inline src/commands/utility/tinyurl.js]: use await and handle errors like this:
```
try {
  const shortUrl = await fetch(`http://tinyurl.com/api-create.php?url=${encodeURIComponent(text)}`)
  channel.send(shortUrl)
} catch (e) {
  throw new CommandError(t('commands:tinyurl.shortingError'))
}
```
you will need of course to import CommandError and create that string.
Also would be nice if you use SwitchEmbed too

File: src/commands/utility/tinyurl.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+const { Command } = require('../../')
+const fetch = require('node-fetch')
+
+module.exports = class Tinyurl extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'tinyurl',
+      aliases: ['short', 'shorturl'],
```
Comment (pedrofracassi) [inline src/commands/utility/tinyurl.js]:
```suggestion
      aliases: ['shorten', 'short', 'shorturl'],
```

File: src/locales/en-US/commands.json [hunk 4]
```diff
@@ -1098,8 +1098,13 @@
...
   "morejpeg": {
-	"commandDescription": "Adds JPEG compression to an image.",
-	"commandUsage": "[image link]",
-	"missingImage": "You need to give me a link or an image!"
+    "commandDescription": "Adds JPEG compression to an image.",
+    "commandUsage": "[image link]",
+    "missingImage": "You need to give me a link or an image!"
+  },
+  "tinyurl": {
+    "commandDescription": "URL Shortener",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]:
```suggestion
    "commandDescription": "Shortens an URL using TinyURL",
```

File: src/locales/en-US/commands.json [hunk 5]
```diff
@@ -1098,8 +1098,13 @@
...
+    "commandUsage": "<url to short>",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]:
```suggestion
    "commandUsage": "<url>",
```

File: src/locales/en-US/commands.json [hunk 6]
```diff
@@ -1098,8 +1098,13 @@
...
+    "missingSentence": "You need to give me a URL to short"
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]:
```suggestion
    "missingSentence": "You need to give me a URL to shorten"
```

**Comment by metehus** [comment]
also use the new URL parameter

**Comment by pedrofracassi** [comment]
Stale. Closing.

**Pull request closed without merging**